plugins/bulk-import/src/components/BulkImportPage.tsx

```diff
@@ -33,21 +36,27 @@ export const BulkImportPage = () => {
     resourceRef: bulkImportPermission.resourceType,
   });
 
+  if (!queryClientRef.current) {
+    queryClientRef.current = new QueryClient();
+  }
+
   const showContent = () => {
     if (bulkImportViewPermissionResult.loading) {
       return <Progress />;
     }
     if (bulkImportViewPermissionResult.allowed) {
       return (
-        <Formik
-          initialValues={initialValues}
-          enableReinitialize
-          onSubmit={async (_values: AddRepositoriesFormValues) => {}}
-        >
-          <FormControl fullWidth>
-            <RepositoriesList />
-          </FormControl>
-        </Formik>
+        <QueryClientProvider client={queryClientRef.current as QueryClient}>
```

Comment:
`as` always trigger me ;)

```suggestion
        <QueryClientProvider client={queryClientRef.current!}>
```